solutions/security/ai/large-language-model-performance-matrix.md

```diff
@@ -41,9 +41,15 @@ Models from third-party LLM providers.
 
 Models you can [deploy yourself](/solutions/security/ai/connect-to-own-local-llm.md).
 
-| **Feature** | - | **Assistant - General** | **Assistant - {{esql}} generation** | **Assistant - Alert questions** | **Assistant - Knowledge retrieval** | **Attack Discovery** |
+| **Feature** | - | **Assistant - General** | **Assistant - {{esql}} generation** | **Assistant - Alert questions** | **Assistant - Knowledge retrieval** | **Attack Discovery** | **Automatic Migration** |
 | --- | --- | --- | --- | --- | --- | --- |
-| **Model** | **Mistral Nemo** | Good | Good | Great | Good | Poor |
-|  | **LLama 3.2** | Good | Poor | Good | Poor | Poor |
-|  | **LLama 3.1 405b** | Good | Great | Good | Good | Poor |
-|  | **LLama 3.1 70b** | Good | Good | Poor | Poor | Poor |
+| **Model** | **Mistral Nemo** | Good | Good | Great | Good | Poor | Poor |
+|  | **LLama 3.2** | Good | Poor | Good | Poor | Poor | Good  |
+|  | **LLama 3.1 405b** | Good | Great | Good | Good | Poor | Poor |
+|  | **LLama 3.1 70b** | Good | Good | Poor | Poor | Poor | Good |
+
+| *Automatic Migration* 
```

Comment:
Does this need to be moved up a line? It's not rendering with the table in the [preview](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/1044/solutions/security/ai/large-language-model-performance-matrix#_open_source_models).